packages/@aws-cdk/aws-iam/lib/oidc-provider.ts

```diff
@@ -110,6 +135,8 @@ export class OpenIdConnectProvider extends Resource implements IOpenIdConnectPro
    */
   public readonly openIdConnectProviderArn: string;
 
+  public readonly openIdConnectProviderIssuer: string;
```

Comment:
Might as well add an `openIdConnectProviderIssuerUrl` property that just exposes `props.url` that was given.

Reply:
I think that at some point we should fix cluster construct to not return the URL or to not return issuer.
because it's different behaviour between console/terraform and cdk. Lets probably leave only issuer here to match original provider interface and try to fix cluster construct later? 
I just really really don't like that cdk has two fields but originally provider and cluster has only one it leads to confusion when using the library

Reply:
Yeah thats ok 👍